src/SingleProductDetailView.css

```diff
@@ -0,0 +1,15 @@
+.single-product-detail-view-wrapper {
+  width: 100%;
+  text-align: center;
+  background-color: azure;
+  margin: 10px;
+}
+
+.single-product-detail-view-container {
+  margin: 10px, auto, 10px, auto;
```

Comment:
check the syntax for the margin rule. this margin isn't being applied right now.